pkg/util/workloads_utils.go

```diff
@@ -212,6 +226,95 @@ func GenRandomStr(length int) string {
 	return rand.SafeEncodeString(randStr)
 }
 
+func PatchUpdateStrategy(c client.Client, object client.Object, updateStrategy map[string]interface{}) error {
+	patch := map[string]interface{}{
+		"spec": map[string]interface{}{
+			"updateStrategy": updateStrategy,
+		},
+	}
+
+	patchByte, _ := json.Marshal(patch)
+	clone := object.DeepCopyObject().(client.Object)
+	return c.Patch(context.TODO(), clone, client.RawPatch(types.MergePatchType, patchByte))
+}
+
+func ReleaseWorkload(c client.Client, object client.Object) error {
+	_, found := object.GetAnnotations()[BatchReleaseControlAnnotation]
+	if !found {
+		klog.V(3).Infof("Workload(%v) is already released", client.ObjectKeyFromObject(object))
```

Comment:
v(5) ?

Reply:
v(3) is appropriate. 
This function is only executed once during rollout.